src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/unnecessarysemicolonintrywithresources/InputUnnecessarySemicolonInTryWithResourcesAllow.java

```diff
@@ -0,0 +1,22 @@
+package com.puppycrawl.tools.checkstyle.checks.coding.unnecessarysemicolonintrywithresources;
+
+import java.io.IOException;
+import java.io.PipedReader;
+import java.io.Reader;
+
+public class InputUnnecessarySemicolonInTryWithResourcesAllow {
```

Comment:
in new generation of inputs, we try to follow good practice - input file is for single configuration.
you did all correctly, but ...

Please add javadoc comment
```
/**
* "allowWhenNoBraceAfterSemicolon" ="false"
*
*/
```

one day in future UTs will take config from input files, and make sure that all "violation" trailing comments are matching to what we have in UT.

Reply:
done